akvo/rsr/filters.py

```diff
@@ -98,26 +98,45 @@ def get_locations(location, locations):
 def location_choices(qs):
     """From a queryset get possible location filter choices"""
 
-    if qs.model is Project:
-        location_model = ProjectLocation
-    elif qs.model is ProjectUpdate:
+    country_ids = (
+        get_recipient_country_ids(qs) if qs.model is Project else
```

Comment:
That was my initial solution, but I removed the project locations because they are not shown in the project description in the project directory.  Do you think we should show them?  

![image](https://cloud.githubusercontent.com/assets/315678/24862959/8a47f6dc-1e1c-11e7-878e-adb389ce46b0.png)